lib/pdksync/configuration.rb

```diff
@@ -43,6 +47,7 @@ def initialize(config_path = ENV['PDKSYNC_CONFIG_PATH'])
       super(@custom_config)
       valid_scm?(git_platform)
       valid_access_token?
+      valid_jenkins_credentials?
```

Comment:
looks like we need to call it in pdksync. Since we dont have to set the variable if the user is not running job through jenkins

Reply:
moved the validation part on utils when jenkins client is created